package.json

```diff
@@ -104,6 +105,7 @@
     "flux": "3.1.3",
     "fuse.js": "3.2.1",
     "get-video-id": "3.1.0",
+    "gfm-code-blocks": "1.0.0",
```

Comment:
How does this affect bundle size?

Reply:
As this package is imported by the `Example` component which is async loaded, only the size of the async chunk has been increased from 4.15 KB (`async-load-design-gutenberg-components`) to 5.6 KB (`async-load-gutenberg-components`):

http://iscalypsofastyet.com/branch?branch=update/devdocs-gutenberg-components